### PR TITLE
Add Video

### PR DESCRIPTION
**Pull request by PradeepSinghMakwana**

### CODE REVIEW

File: beginner_source/blitz/autograd_tutorial.py [hunk 1]
```diff
@@ -3,6 +3,14 @@
 Autograd: Automatic Differentiation
 ===================================
 
+**Video By**: `Elliot Waite`_
+
+.. raw:: html
+
+   <div style="margin-top:10px; margin-bottom:10px;">
+     <iframe width="560" height="315" src="https://www.youtube.com/embed/MswxJw-8PvE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (elliotwaite) [inline beginner_source/blitz/autograd_tutorial.py:11]: Maybe remove the "autoplay; " part. That was the only difference I noticed between this code and this pull request where a video was added to the top of the 60 Min Blitz Tutorial: https://github.com/pytorch/tutorials/commit/fb00d7827a7d7646390f6dc484684f303bd76789

**Comment by elliotwaite** [comment]
These error messages in the [log file](https://app.netlify.com/sites/pytorch-tutorials-preview/deploys/5ee36238800a520008ff227a) do not seem very specific:
```
4:10:46 AM: Failing build: Failed to prepare repo
4:10:47 AM: failed during stage 'preparing repo': exit status 1
```

It may have failed for a reason unrelated to the changed file. Perhaps you could try rerunning the checks. Do you see an option to do that?

File: beginner_source/blitz/autograd_tutorial.py (hunk 1 above)
Comment (PradeepSinghMakwana) [reply beginner_source/blitz/autograd_tutorial.py]: Thanks. You crushed the bug, without any error logs. All tests passed, but still I cannot see the changes on [this](https://pytorch.org/tutorials/beginner/blitz/autograd_tutorial.html) page. I think the changes will take effect in the next release.

Comment (PradeepSinghMakwana) [reply beginner_source/blitz/autograd_tutorial.py]: Okay. I see the changes need to be approved. This is unmerged pull request until then.

**Comment by elliotwaite** [comment]
@PradeepSinghMakwana, it looks like this pull request might be able to be merged soon, but I think you'll have to sign Facebook's CLA (contributor license agreement) first, as mention by the @facebook-github-bot above. Here's the link again to where you can sign it: https://code.facebook.com/cla. Let me know if you have any questions about this. I recently also signed the CLA myself for a different pull request I submitted.

**Comment by PradeepSinghMakwana** [comment]
@ElliotWaite Thanks. But, I am sorry. I broke my leg. It will take me
almost 1-2 week to reach my computer. I will try my best. I will also sign
for Facebook CLA

**Comment by elliotwaite** [comment]
@PradeepSinghMakwana, oh dang, sorry to hear about your broken leg. I hope you feel better.

No worries about the delay, whenever you get around to it is fine with me. Thanks for the update.

**Comment by PradeepSinghMakwana** [comment]
@elliotwaite Thanks. I am back. I signed for the Facebook CLA, but still windows test is failing. I don't see anything specific to my pull request. Can you help?

**Comment by elliotwaite** [comment]
Hey @PradeepSinghMakwana, glad to hear you're back.

It looks like the failure was unrelated to the changes in this pull request. I've encountered this situation before and just rerunning the checks at a later time resolved the issue for me. I would suggest trying to just rerun the checks. I forget how I reran the checks, but you can see if any of the suggestions here will work for you: https://stackoverflow.com/questions/52408592/how-to-relaunch-github-check-without-pushing-new-commits/52412240

**Comment by elliotwaite** [comment]
Hmm, looks like the same error is still occurring. Maybe's there's currently an issue with the CI. I'd suggest just retrying the checks at another time. Maybe after a few days.

**Comment by elliotwaite** [comment]
@PradeepSinghMakwana, looks like all the checks have finally passed! Thanks for following up on this.

Hopefully, someone will now be able to review it.

**Comment by elliotwaite** [comment]
This PR can be closed. It was replaced by: https://github.com/pytorch/tutorials/pull/2625

**Pull request closed without merging**